lib/src/ui.dart

```diff
@@ -192,12 +192,19 @@ class _CropState extends State<Crop> with TickerProviderStateMixin {
     _startOffset = widget.controller._offset;
     _endOffset = widget.controller._offset;
 
+    /* details.rotation is in radians, 
+       so using appropriate setter for rotation
+       This will enable rotation using gesture
+    */
+    widget.controller.rotationRads =  details.rotation;
```

Comment:
Here, this is the code which reads the rotation expressed from gesture detector's callback and sets to our controller